[Database] TAPP-111: Remove pay1 and pay2 from Assignments and Offers

michellemtchai opened the pull request:
Please refer to issue #111

dingtaoliu commented:
There is a merge conflict with the schema file because I just merged in a pr that included migrations which changed the schema version.  It seems like we will have this problem every time there are parallel branches that modify the schema, should we just remove the schema file from version control?

michellemtchai commented:
I think we should if `schema.rb` can generated by rails instead of just be append to by it. Also, I noticed that when you switch from a branch that has deleted table a to a branch that still contain table a, table a doesn't come back on the second branch. This may become a bigger issue later on. Should I make another issue for that?

michellemtchai commented:
I've fixed the conflict.

dingtaoliu commented:
For the issue that you brought up, the problem is that the migrations are out of sync between the two branches.  I think we can fix this without resetting the entire database by dropping the 'schema_migrations' table that keeps track of migrations that have been run.  After this, we can run db:migrate again to get the db to the state we want it to be in a particular branch.

George334456 commented on api/db/schema.rb:
```diff
@@ -10,7 +10,11 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
+<<<<<<< HEAD
```
Is this supposed to be here?

michellemtchai replied:
No, it's not supposed to be here. Great catch!

George334456 reviewed (changes requested):
I think you left something inside, unless I'm looking at the wrong thing.

michellemtchai committed:
fix(db): remove pay1 and pay2 from assignments and offers
remove pay1 and pay2 columns from assignments and offers table due to being unused

michellemtchai commented:
I've removed the leftover line you caught.

George334456 reviewed (approved):
Good stuff
